### PR TITLE
Update readme config reload

### PR DESCRIPTION
**Pull request by amfern**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ but be aware that version will not be updated.
 
       map Alt+Insert kitten kitty_grab/grab.py
 
-* Restart Kitty.
+* Reload config with Ctrl+Shift+f5 or Restart Kitty.
```
Comment (yurikhan) [inline README.md:34]: This assumes default configuration. I’d prefer to say something that acknowledges that the user might have changed the default, like “Reload config (Ctrl+Shift+F5 by default, see `load_config_file`)” (hyperlink to the Kitty configuration docs).

Comment (yurikhan) [reply README.md:34]: Also, keys should be formatted in monospace, see mentions of other keys throughout the file.

Comment (yurikhan) [reply README.md:34]: Overall, thanks for reminding me Kitty now supports config reloading. I think there is at least one more mention of restart down the file, that should probably be changed too.

**Comment by amfern** [comment]
Thanks for review, i have changed the readme, please see if that what you had in mind

**Comment by amfern** [comment]
Updated.
Would you like me to squash the commits?

File: README.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ or rebind previous/next tab to different keys in `kitty.conf`
     map Ctrl+Page_Up     previous_tab
     map Ctrl+Page_Down   next_tab
 
-(Remember to [reload config](https://sw.kovidgoyal.net/kitty/conf/) if you modify `kitty.conf`.)
+(Remember to [reload config](https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Reload-kitty.conf/) if you modify `kitty.conf`.).
```
[the remark this answers is not in the document]
Comment (amfern) [reply README.md]: No problem at all, i like that you have standartds.

**Comment by yurikhan** [comment]
Thank you.

**Pull request merged**